### PR TITLE
add curl

### PR DESCRIPTION
**Pull request by ran1995ml**
Fixes #XXXX.

#### Release note

<hr>

##### Key changed/added classes in this PR
 * `MyFoo`
 * `OurBar`
 * `TheirBaz`

<hr>

This PR has:

- [ ] been self-reviewed.
   - [ ] using the [concurrency checklist](https://github.com/apache/druid/blob/master/dev/code-review/concurrency.md) (Remove this item if the PR doesn't have any relation to concurrency.)
- [ ] added documentation for new or modified features or behaviors.
- [ ] a release note entry in the PR description.
- [ ] added Javadocs for most classes and all non-trivial methods. Linked related entities via Javadoc links.
- [ ] added or updated version, license, or notice information in [licenses.yaml](https://github.com/apache/druid/blob/master/dev/license.md)
- [ ] added comments explaining the "why" and the intent of the code wherever would not be obvious for an unfamiliar reader.
- [ ] added unit tests or modified existing tests to cover new code paths, ensuring the threshold for [code coverage](https://github.com/apache/druid/blob/master/dev/code-review/code-coverage.md) is met.
- [ ] added integration tests.
- [ ] been tested in a test Druid cluster.

### CODE REVIEW

File: distribution/docker/Dockerfile [hunk 1]
```diff
@@ -86,6 +91,7 @@ COPY distribution/docker/druid.sh /druid.sh
 COPY distribution/docker/peon.sh /peon.sh
 COPY distribution/docker/deduplicate_jars.sh /deduplicate_jars.sh
 
+
```
Comment (LakshSingla) [inline distribution/docker/Dockerfile:94]: nit: unnecessary

**Review by LakshSingla — COMMENTED** [review]
Why is cURL required in the Docker image?

**Comment by abhishekagarwal87** [comment]
Why do you need curl in this image and how much does the size increase post adding curl?

**Comment by FrankChen021** [comment]
I think the size increase is not a problem based on the changes, but please elaborate more about the motivation. 

Usually, if it's used for one-time debugging, we install it during runtime instead of putting it in the image

**Pull request closed without merging**